packages/graphql-toolbox/README.md

```diff
@@ -67,6 +68,12 @@ This will pre-fill the login window input fields `Username` with `testuser` and
 
 Note: The plus symbol (+) in the connectURL needs to be URL encoded to become `%2B`. For example `bolt+s` becomes `bolt%2Bs`.
 
+The URL parameter `pw` is additionally available to provide the database password.
```

Comment:
Any particular reason for the password being a separate parameter?
https://datatracker.ietf.org/doc/html/rfc1738#section-3.1

Reply:
Mm missed this one. Will refactor

Reply:
Cool.
Don't forget to update the description of this PR as well after the changes 🚋